package.json

```diff
@@ -2,26 +2,26 @@
   "name": "root",
   "description": "Monorepo for chrono packages",
   "engines": {
-    "node": ">=22.14.0",
-    "npm": ">=10.9.2"
+    "node": ">=20.18.3",
+    "pnpm": ">=10.6.2"
   },
   "scripts": {
     "clean": "pnpm --recursive run clean",
     "build": "pnpm --recursive run build",
     "typecheck": "pnpm --recursive run typecheck",
     "check": "biome check",
     "format": "biome format --write",
-    "lint": "biome lint --write",
+    "lint": "biome lint",
```

Comment:
technically biome check does biome format + biome lint

Thats why i made lint just do the write. But might be confusing so im fine with this